pkg/config/linters_settings.go

```diff
@@ -49,6 +49,7 @@ var defaultLintersSettings = LintersSettings{
 	},
 	Gofumpt: GofumptSettings{
 		LangVersion: "",
+		ModulePath:  "",
```

Comment:
```suggestion
```

Reply:
I think we should be consistent, either we add defaults like `LangVersion` or we don't.

Reply:
Why is `LangVersion` specified with the default value but we shouldn't specify `ModulePath`?

Reply:
I would have the same question. I'm not strongly opinionated on supplying the blank string or omitting the property entirely as long as it's consistent.

Reply:
Sorry I was on my phone and the diff was a bit weird, you can keep it.